README.md

```diff
@@ -1,9 +1,9 @@
 <p align="center">
     <a href="https://github.com/yiisoft" target="_blank">
-        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
+        <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px" alt="Yii">
     </a>
-    <a href="https://bulma.io/" target="_blank" rel="external">
-        <img src="docs/images/bulma-logo.png" height="100px">
+    <a href="https://bulma.io/" target="_blank">
+        <img src="https://bulma.io/assets/brand/Bulma%20Logo.svg" height="80px" alt="Bulma">
```

Comment:
It looks strange: 

![image](https://github.com/yiisoft/yii-bulma/assets/525501/73690e17-e270-4685-992b-becf65702124)

Reply:
Yes! :))
Choose one https://bulma.io/brand/

Reply:
Need to choose image that will look good in both dark and light theme. Also need to add this image to repo.

Reply:
removed